### DISCUSSION TITLE
how do i put multiple commands in entrypoint?

### DISCUSSION THREAD

**Discussion by imperialguy** [containers/podman#13028] [General]
I am looking for something like this: 
```
podman run --entrypoint "/sbin/init && /path/to/my/script.sh"
```

**Comment by rhatdan** [comment]
You don't.  That makes no sense.  If you are going to use systemd, then add your script.sh as a unit file to be started by systemd at boot.

**Reply by imperialguy** [reply]
I know, but I didn't pick that route because my script is not exactly a service. And systemd unit files are usually meant for services. 

If you are saying there's no other choice, then I am gonna have to treat my script as a service and make a unit file for it. But, I was wondering if someone has figured out an alternative solution already.

**Reply by mheon** [reply]
While Podman does support multi-part entrypoints (e.g. `--entrypoint '["/foo/bar", "arg1", "arg2"]'`) this is only for specifying a single command with multiple parts, IE path to a command and arguments. You cannot use an entrypoint to start 2 distinct commands; there is no way in Podman to do this at present, outside of a separate `podman exec` command run after the container is started.

**Comment by rhatdan** [comment]
You can setup a one time unit file to run what every you want.  No reason to be a service.  There is a lot of stuff in unit files that just init the system or container.
